torch/autograd/_functions/pointwise.py

```diff
@@ -284,17 +326,14 @@ def backward(ctx, grad_output):
 class _ConstantGrad(Function):
     grad_value = 0
 
-    def __init__(self, *args):
-        super(_ConstantGrad, self).__init__()
-        self.args = args
+    @classmethod
+    def forward(cls, ctx, *args):
+        ctx._num_args = len(args)
+        return getattr(args[0], cls.__name__.lower())(*args[1:])
 
-    def forward(self, i):
-        return getattr(i, type(self).__name__.lower())(*self.args)
-
-    def backward(self, grad_output):
-        grad_input = grad_output.new(*repeat(1, grad_output.dim()))
-        grad_input = grad_input.fill_(self.grad_value).expand_as(grad_output)
-        return grad_input.mul(grad_output)
+    @classmethod
```

Comment:
I think I use cls in both methods  (in forward for the `cls.__name__`, in backward for the `cls.grad_value`).